src/main/java/duke/ui/MainWindow.java

```diff
@@ -0,0 +1,70 @@
+package duke.ui;
+
+import duke.Duke;
+import duke.command.CommandResponse;
+import javafx.application.Platform;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
```

Comment:
I like the lexicographic way in which you arrange your import statements in all your classes, maybe it will look neater if there is a line break in between the the import statements at line 4 and 5?

Reply:
Thank you for the suggestions! I will look into them.